docs/modules/ROOT/pages/auth/index.adoc

```diff
@@ -1,51 +0,0 @@
-[[auth]]
```

Comment:
(It's not here)
We probably also need to clean up the `content-nav.adoc` file

Reply:
Great spot, thank you!